cli/cmd/build.go

```diff
@@ -19,6 +20,18 @@ var buildCmd = &cobra.Command{
 	Short:        "Build docker images",
 	Long:         "Build docker images using docker compose",
 	SilenceUsage: true,
+	PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+		checklist := util.NewValidationCheckList()
+		return util.ValidateEnvironment(cmd.Context(),
+			[]util.ValidationCallback{
+				checklist.DockerEngineRunning,
+				checklist.MinDockerComposeVersion,
+				checklist.DockerInstalled,
+				checklist.TerraformInstalled,
+				checklist.AwsInstalled,
+				checklist.AwsSessionManagerPluginInstalled,
+			})
```

Comment:
if you change the signature of this method to be varadic, you can do something like this:

```suggestion
				checklist.DockerEngineRunning,
				checklist.MinDockerComposeVersion,
				checklist.DockerInstalled,
				checklist.TerraformInstalled,
				checklist.AwsInstalled,
				checklist.AwsSessionManagerPluginInstalled,
			)
```

which I think reads a little easier